modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -163,7 +163,6 @@ apiTemplateFiles are for API outputs only (controllers/handlers).
     protected Map<String, String> reservedWordsMappings = new HashMap<String, String>();
     protected String templateDir;
     protected String embeddedTemplateDir;
-    protected String commonTemplateDir = "_common";
```

Comment:
this is now handled by template loading functionality, as that is not the responsibility of DefaultCodegen.